### PR TITLE
W-10475111 create robot locators for api 54 (236)

### PR DESCRIPTION
**Pull request by iskander-sfdc**
# Critical Changes

# Changes

# Issues Closed

# New Metadata

## Unpackaged Metadata

# Deleted Metadata

# Testing Notes
create an or using `qa_org` flow
run stable robot tests
```
cci task run robot --suites robot/EDA/tests --exclude unstable
```
Should work on 234 and 236 orgs

### CODE REVIEW

File: robot/EDA/resources/locators_53.py [hunk 1]
```diff
@@ -0,0 +1,414 @@
+""" Locators for Winter '22 """
```
Comment (salvadorsfdc) [inline robot/EDA/resources/locators_53.py:1]: I think this needs to be something like:
```
from locators_54 import *
import copy
eda_lex_locators = copy.deepcopy(eda_lex_locators)
#Override the locator selector value
eda_lex_locators['eda_settings_new']['dropdown_input']="//label[text()='{}']/../descendant::input[@role='combobox']"
```

**Review by salvadorsfdc — CHANGES_REQUESTED** [review]

File: robot/EDA/resources/locators_53.py (hunk 1 above)
Comment (iskander-sfdc) [reply robot/EDA/resources/locators_53.py:1]: Committed this change. Robot tests in meta ci are passing

File: robot/EDA/resources/EDA.robot [hunk 2]
```diff
@@ -135,11 +135,11 @@ Get Affiliation Mappings
     ${NS} =                 Get EDA namespace prefix
     [Arguments]             ${field_name}
     ${result} =             SOQL Query
-    ...                     SELECT ${NS}${field_name} FROM ${NS}Affl_Mappings__c
```
Comment (salvadorsfdc) [inline robot/EDA/resources/EDA.robot:138]: I'm not sure about removing the NS variable from here @jonathanjayaraj @iskander-sfdc . It seems it was working fine previously.

File: robot/EDA/resources/EDA.robot [hunk 3]
```diff
@@ -135,11 +135,11 @@ Get Affiliation Mappings
...
+    ...                     SELECT ${field_name} FROM ${NS}Affl_Mappings__c
     ${count} =              Get Affl Records Count       ${field_name}
     @{ValueList}=           Create List
     FOR     ${index}        IN RANGE    ${count}
-        ${name} =               Set variable    ${result}[records][${index}][${NS}${field_name}]
```
Comment (salvadorsfdc) [inline robot/EDA/resources/EDA.robot:142]: I'm not sure about removing the NS variable from here. It seems it was working fine previously.

File: robot/EDA/resources/EDA.robot [hunk 4]
```diff
@@ -174,7 +174,7 @@ Get Affl Records Count
     ${NS} =                 Get EDA namespace prefix
     [Arguments]             ${field_name}
     ${result} =             SOQL Query
-    ...                     SELECT COUNT(${NS}${field_name}) FROM ${NS}Affl_Mappings__c
+    ...                     SELECT COUNT(${field_name}) FROM ${NS}Affl_Mappings__c
```
Comment (salvadorsfdc) [inline robot/EDA/resources/EDA.robot:177]: I'm not sure about removing the NS variable from here. It seems it was working fine previously.

File: robot/EDA/resources/EDA.robot (hunk 2 above)
Comment (jonathanjayaraj) [reply robot/EDA/resources/EDA.robot:138]: Correct. we'll need the ${NS} variable to add the EDA namespace if it is in a managed org. This was working previously.I'm about to try it in a regression org which is managed.

Comment (iskander-sfdc) [reply robot/EDA/resources/EDA.robot:138]: Yes, this change fixes this test for a regression org which uses managed package

**Review by salvadorsfdc — APPROVED** [review]
looks good

File: robot/EDA/resources/EDA.robot (hunk 4 above)
Comment (salvadorsfdc) [reply robot/EDA/resources/EDA.robot:177]: just for reference the field_name in this case is name which doesnt require namespace

**Pull request merged**